jobs/rocoto/awips_20sh.sh

```diff
@@ -19,14 +19,13 @@ source "${HOMEgfs}/ush/load_fv3gfs_modules.sh"
 status=$?
 (( status != 0 )) && exit "${status}"
 
-export job="awips"
+export job="awips_20sh.sh"
```

Comment:
`job` is not a shell script.  It is the name of the job.
```suggestion
export job="awips_20km"
```
Please consider renaming this script to `awips_20km.sh`